common/logger/src/libra_logger.rs

```diff
@@ -336,9 +336,9 @@ impl Writer for FileWriter {
 }
 
 /// Converts a record into a string representation:
-/// UNIX_TIMESTAMP LOG_LEVEL FILE:LINE MESSAGE
+/// UNIX_TIMESTAMP LOG_LEVEL FILE:LINE [thread_name] MESSAGE JSON_DATA
 /// Example:
-/// 2020-03-07 05:03:03 INFO common/libra-logger/src/lib.rs:261 Hello
+/// 2020-03-07 05:03:03 INFO common/libra-logger/src/lib.rs:261 [thread_name] Hello { "world": true }
```

Comment:
btw I think the order of timestamp and level is wrong

Reply:
I've fixed it so the timestamp is at the beginning.